package.json

```diff
@@ -33,12 +33,12 @@
     "react-tooltip": "^3.6.0",
     "redux": "^4.0.0",
     "redux-logger": "^3.0.6",
-    "redux-mock-store": "^1.5.1",
+    "redux-mock-store": "^1.5.3",
     "redux-test-utils": "^0.2.2",
     "redux-thunk": "^2.2.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "set NODE_ENV=dev && react-scripts start",
```

Comment:
I don't know if the new addition to line 41 is necessary, apart from that, PR looks good to me.

Reply:
I had put it to explicitly state the environment.
I have since removed it because the environment without it being put in the script